lms/models/task_done.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, DateTime, Integer, UnicodeText, text
+
+from lms.db import BASE
+from lms.models._mixins import CreatedUpdatedMixin
+
+
+class TaskDone(CreatedUpdatedMixin, BASE):
+    __tablename__ = "task_done"
+
+    id = Column(Integer, autoincrement=True, primary_key=True)
+    key = Column(UnicodeText, nullable=False, unique=True)
+    expires_at = Column(
+        DateTime, nullable=False, server_default=text("now() + interval '30 days'")
+    )
```

Comment:
A future PR will add a periodic task that deletes expired `task_done` rows, so rows will get deleted after 30 days by default. This is just to prevent the table from growing forever. Note that this does not mean that any matching emails will be considered duplicates for 30 days because the `key` that's generated below includes a date in the key.

Reply:
Added an issue to keep track of this: https://github.com/hypothesis/lms/issues/5348